web/src/features/moreCast2/util.ts

```diff
@@ -102,6 +102,10 @@ export const validForecastPredicate = (row: MoreCast2Row) =>
   !isUndefined(row.windSpeedForecast) &&
   !isNaN(row.windSpeedForecast.value)
 
+export const isForecastRow = (row: MoreCast2Row) => {
+  return !rowContainsActual(row) && !isPreviousToToday(row.forDate)
```

Comment:
How do you feel about renaming `isPreviousToToday` to `isBeforeToday`?

Reply:
I feel slightly embarrassed that it was ever named `isPreviousToToday` in the first place haha. I'll definitely rename it

Reply:
Lol no worries